src/pipecat/processors/frame_processor.py

```diff
@@ -234,6 +235,7 @@ def __init__(
         self._register_event_handler("on_after_process_frame", sync=True)
         self._register_event_handler("on_before_push_frame", sync=True)
         self._register_event_handler("on_after_push_frame", sync=True)
+        self._register_event_handler("on_error", sync=True)
```

Comment:
Do we want this to be sync=True? This means we won't create a task for it. Since it's an error this might be fine, I haven't thought much about it.